virttest/libvirt_remote.py

```diff
@@ -44,3 +44,22 @@ def update_remote_file(params, value,
     except Exception as err:
         raise exceptions.TestFail("Unable to update {}: {}"
                                   .format(file_path, err))
+
+
+def delete_remote_file(file_path, params):
+    """
+    Delete file on remote
+
+    :param file_path: The file to be deleted
+    :param params: Test params
+    """
+    remote_ip = params.get("server_ip", params.get("remote_ip"))
+    remote_pwd = params.get("server_pwd", params.get("remote_pwd"))
+    remote_user = params.get("server_user", params.get("remote_user"))
+    if not all([remote_ip, remote_pwd, remote_user]):
+        raise exceptions.TestError("remote_[ip|user|pwd] are necessary!")
+    remote_session = remote.wait_for_login('ssh', remote_ip, '22',
+                                           remote_user, remote_pwd,
+                                           r"[\#\$]\s*$")
+    remote_session.cmd("rm -rf %s" % file_path, ignore_all_errors=True)
```

Comment:
This is very useful! However, are we considering returning status? Like, True if the removal succeeds, False if it fails. Maybe a simple check after command executed? That way, we users will know whether the file is actually removed or no. Just my opinion.

Reply:
Sure, updated.